pkg/config/utils/tags.go

```diff
@@ -5,7 +5,34 @@
 
 package utils
 
-import "github.com/DataDog/datadog-agent/pkg/config"
+import (
+	"sort"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// removeDuplicatesAndSort returns a new array, sorted without duplicates.
+func removeDuplicatesAndSort(elements []string) []string {
```

Comment:
golang provide now a generic [Unique](https://github.com/mpvl/unique/blob/cbe035fff7de/unique.go#L17) [¶](https://pkg.go.dev/github.com/mpvl/unique#Unique) function

maybe we can use it.